.gitignore

```diff
@@ -1,5 +1,6 @@
 # Executables
 bin/
+spicetify-cli
```

Comment:
```suggestion
spicetify
```
![image](https://user-images.githubusercontent.com/77577746/210908591-5d348c88-5578-4608-be4a-c49de2052781.jpeg)

Reply:
add both instances then...

```suggestion
spicetify-cli
spicetify
```

Reply:
Maybe we can have it ignore both, as either one of them could be what's generated depending on whether someone read through the instructions or not

Reply:
Ririxi got to it first lol